src/listener.js

```diff
@@ -153,12 +153,12 @@ function _forwardEvent(event_object) {
 
     //logging info to the console
     console.log(fonts.info(
-      fonts.highlight(`${event_object.resource}:${event_object.event}`) +
-      ' received'));
+        fonts.highlight(`${event_object.resource}:${event_object.event}`) +
+        ' received'));
 
     //gathering some details
     const options = {
-        hostname: 'localhost',
+        hostname: specifications.target,
```

Comment:
@legion49f what happens if the TARGET is not set in the environmental variables? Whatever it's value, it's now hardcoded in the `specifications` object, even if it is undefined. So maybe let's add a case before changing the hostname in the options so that it only sets it to `specifications.target` if `specifications.target` is defined. Otherwise it can just leave it as 'localhost'.

Reply:
ok i will get this changed and push later today/tomorrow

Reply:
ok I set the target as optional.
```js
 if (!specs.target) {
        specs.target = 'localhost'
    }

Reply:
ok i fixed this issue and took the time to test everything manually to make sure it works as expected

Reply:
LGTM